core/src/main/java/org/apache/calcite/util/BuiltInMethod.java

```diff
@@ -628,6 +628,7 @@ public enum BuiltInMethod {
   IS_EMPTY(Collection.class, "isEmpty"),
   SUBMULTISET_OF(SqlFunctions.class, "submultisetOf", Collection.class,
       Collection.class),
+  ARRAY_COMPACT(SqlFunctions.class, "removeNulls", List.class),
```

Comment:
this could also be named `compact` so it's easier to find, removeNulls is also ok.

Reply:
+1, think the BuiltInMethod name normally matches the SqlFunctions name

Reply:
fixed @MasseGuillaume @tanclary